core/src/main/java/com/scalar/db/storage/multistorage/MultiStorageConfig.java

```diff
@@ -81,14 +69,18 @@ private void loadDatabaseConfigs() {
     ImmutableMap.Builder<String, DatabaseConfig> builder = ImmutableMap.builder();
     for (String storage : storages.split(",", -1)) {
       Properties dbProps = new Properties();
-      for (String propertyName : props.stringPropertyNames()) {
+      for (String propertyName : getProperties().stringPropertyNames()) {
         if (propertyName.startsWith(STORAGES + "." + storage + ".")) {
           dbProps.put(
               propertyName.replace("multi_storage.storages." + storage + ".", ""),
-              props.getProperty(propertyName));
+              getProperties().getProperty(propertyName));
         }
       }
 
+      // since operations are copied in MultiStorage, we don't need to copy operations in the
+      // storage
+      dbProps.put(DatabaseConfig.NEED_OPERATION_COPY, "false");
```

Comment:
If we use Multi Storage, we set `scalar.db.need_operation_copy` to `false` to avoid to copy operation instances twice.